.neutrinorc.js

```diff
@@ -1,37 +1,127 @@
 const { join } = require('path');
 const merge = require('deepmerge');
-
-require('babel-register')({
-  plugins: [
-    [require.resolve('babel-plugin-transform-es2015-modules-commonjs'), {
-      useBuiltIns: true
-    }],
-    require.resolve('babel-plugin-transform-object-rest-spread'),
-  ],
-  cache: false,
-});
-
-const theme = require('./src/theme').default;
+const MODULES = join(__dirname, 'node_modules');
 
 module.exports = {
   use: [
-    ['neutrino-preset-mozilla-frontend-infra/styleguide', {
-      components: 'src/components/**/index.jsx',
-      theme: theme.styleguide,
-      styles: {
-        StyleGuide: theme.styleguide.StyleGuide,
-      },
-      editorConfig: {
-        theme: 'material',
-      },
-      showUsage: true,
-      skipComponentsWithoutExample: false,
-      styleguideComponents: {
-        Wrapper: join(__dirname, 'src/styleguide/ThemeWrapper.jsx'),
-        StyleGuideRenderer: join(__dirname, 'src/styleguide/StyleGuideRenderer.jsx'),
+    ['@neutrinojs/airbnb', {
+      eslint: {
+        parserOptions: {
+          ecmaFeatures: {
+            legacyDecorators: true
+          }
+        },
+        emitWarning: process.env.NODE_ENV === 'development',
```

Comment:
We'll likely be making this change upstream soon, see:
https://github.com/neutrinojs/neutrino/issues/1131#issuecomment-428982103